content/src/content/docs/learn/tutorials/calling-an-api.mdx

```diff
@@ -0,0 +1,452 @@
+---
+type: Tutorial
+title: Calling an API
+tags:
+  - some
+  - tags
+sidebar:
+  order: 4
+---
+
+import { Aside } from "@astrojs/starlight/components"
+
+Next up we're going to take a look at an endpoint that makes an API call to
+another service. Here's the code:
+
+```ts twoslash title=index.ts ins={9-19} showLineNumbers=false
+import express from "express"
+
+import { NodeHttpServer, NodeRuntime } from "@effect/platform-node"
+import { HttpRouter } from "@effect/platform"
+import { Effect } from "effect"
+
+const app = express()
+
+app.get("/query", async (req, res) => {
+  const query = req.query.q
+  if (typeof query !== "string") {
+    res.status(400).send("Query parameter 'q' is required")
+    return
+  }
+
+  const url = `https://en.wikipedia.org/w/rest.php/v1/search/page?q=${query}&limit=1`
```

Comment:
```suggestion
  const baseUrl = "https://en.wikipedia.org/w/rest.php/v1/search/page`
  const queryParams = `?q=${query}&limit=1`
  const url = `${baseUrl}${queryParams}`
```
To avoid horizontal scroll when the screen is wide enough to avoid it.